static/js/pages/instances/CreateInstanceForm.tsx

```diff
@@ -59,85 +75,103 @@ const CreateInstanceForm: FC = () => {
     instanceType: Yup.string().required("Instance type is required"),
   });
 
-  interface FormValues {
-    name: string;
-    image: RemoteImage | undefined;
-    instanceType: string;
-    profiles: string[];
-    yaml?: string;
-    unchangedYaml?: string;
+  function notifyInProgress() {
+    instanceListNotify?.info(
+      <>A new instance has successfully been requested.</>,
+      "Instance creation"
+    );
   }
 
-  const closeAndNotifySuccess = (instanceName: string, action: string) => {
-    void queryClient.invalidateQueries({
-      queryKey: [queryKeys.instances],
-    });
+  function notifyLaunchedAndStarted(instanceLink: ReactNode) {
     instanceListNotify?.success(
-      <>
-        Instance{" "}
-        <Link to={`/ui/${panelParams.project}/instances/${instanceName}`}>
-          {instanceName}
-        </Link>{" "}
-        {action}.
-      </>
+      <>Instance {instanceLink} successfully launched and started.</>
```

Comment:
Here and everywhere else - maybe it's better to extend the type for the first argument to `string | ReactNode`, so you don't have to wrap it with a fragment every time?

Reply:
The `instanceLink` used inside is a `ReactNode`, so we'd still need the fragment wrapper.